src/node_errors.cc

```diff
@@ -141,7 +141,7 @@ static std::string GetErrorSource(Isolate* isolate,
 }
 
 void PrintStackTrace(Isolate* isolate, Local<StackTrace> stack) {
-  for (int i = 0; i < stack->GetFrameCount() - 1; i++) {
+  for (int i = 0; i < stack->GetFrameCount(); i++) {
```

Comment:
Is this still needed?

Reply:
Yes, it is. I believe it's unexpected behavior.